TRADEOFFS.md

```diff
@@ -0,0 +1,18 @@
+# Tradeoffs
+
+
+## Potential reasons to use React
+
+- React has a HUGE communitity! If you have a problem or need odds are you'll be able to find someone who shares your needs and may already have a solution!
+- Create React App! Is a great way to start a React project without needing to futz around with tooling.
+
+## Potential reasons to not use React
+
+- Most React apps typically require a build step, this isn't technically required, but you'll find that the vast majority of React apps have one, and all learning material assumes one.
```

Comment:
However, the overwhelmingly majority of serious web projects will have _a_ build step anyway, if only to minify code 🤔